example/ios/Podfile.lock

```diff
@@ -315,10 +318,14 @@ PODS:
   - React-logger (0.71.1):
     - glog
   - react-native-airship (16.0.1):
-    - AirshipFrameworkProxy (= 3.0.2)
+    - AirshipFrameworkProxy (= 4.1.0)
     - React-Core
-  - react-native-safe-area-context (4.7.1):
+  - react-native-safe-area-context (4.5.0):
```

Comment:
Did we want to downgrade this and the couple other dependencies below? Not sure if it makes much difference either way 🤷

Reply:
These are dependencies with a caret operator (^):
``` json
"react-native-gesture-handler": "^2.8.0",
"react-native-safe-area-context": "^4.4.1",
"react-native-screens": "^3.18.2",
```
I'm surprised `yarn install` get those versions for me if there are higher versions... But since it's just for the Podfile.lock I think it's safe to say it's no big deal (I'll confirm that with mouna or ogy tomorrow).

Reply:
I confirmed with mouna it's weird but it's fine